### PR TITLE
Dx 940 943 PHP and Ruby hosted messaging docs

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-940.s3-website-us-east-1.amazonaws.com/

**Review by dtolb — CHANGES_REQUESTED** [review]
For the spots where we have the HTTP Response, can you add the PHP/Ruby console.

File: numbers/guides/hostedMessaging.md [hunk 1]
```diff
@@ -157,6 +180,23 @@ Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
 
 ```
 
+{% sample lang="php" %}
+
+```php
+print_r($account->checkTnsPortability(array("5554443333", "5553334444"))->ImportTnCheckerPayload);
```
Comment (dtolb) [inline numbers/guides/hostedMessaging.md]: Can you break this out to an example more like the ruby where the response is stored as a variable

File: numbers/guides/hostedMessaging.md [hunk 2]
```diff
@@ -427,6 +519,19 @@ Content-Type: application/xml; charset=utf-8
 </ImportTnOrder>
 ```
 
+{% sample lang="php" %}
+
+```php
+print_r($account->getImportTnOrder("order_id"));
```
Comment (dtolb) [inline numbers/guides/hostedMessaging.md]: Same as above, break out to assign as variable

File: numbers/guides/hostedMessaging.md [hunk 3]
```diff
@@ -540,6 +657,19 @@ Content-Type: application/xml; charset=utf-8
 </TNs>
 ```
 
+{% sample lang="php" %}
+
+```php
+print_r($account->getInserviceNumbers());
```
Comment (dtolb) [inline numbers/guides/hostedMessaging.md]: Same as above, break out to assign as variable

**Review by dtolb — CHANGES_REQUESTED** [review]
A few small changes with print statements

**Comment by jmulford-bw** [comment]
> For the spots where we have the HTTP Response, can you add the PHP/Ruby console.

What content should go in there? The response object?

File: numbers/guides/hostedMessaging.md (hunk 3 above)
Comment (jmulford-bw) [reply numbers/guides/hostedMessaging.md]: done 👍

File: numbers/guides/hostedMessaging.md (hunk 2 above)
Comment (jmulford-bw) [reply numbers/guides/hostedMessaging.md]: done 👍

File: numbers/guides/hostedMessaging.md (hunk 1 above)
Comment (jmulford-bw) [reply numbers/guides/hostedMessaging.md]: done 👍

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-940.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-940.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-940.s3-website-us-east-1.amazonaws.com/

**Pull request merged**